migrator/migrator.go

```diff
@@ -100,19 +103,42 @@ func (m *migrator) applyMigrations(files []file.File, up bool, steps int, timeou
 	return needsMigration, nil
 }
 
-func chooseMigrations(files []file.File, alreadyMigrated map[int64]bool, up bool, steps int) []file.File {
+func maxVersion(alreadyMigrated map[int64]bool) int64 {
+	result := int64(0)
+	for version, isMigrated := range alreadyMigrated {
+		if isMigrated && version > result {
```

Comment:
What is this `isMigrated` for (it is always `true`)? Why we have` map[int64]bool` type at all? I think we should use `map[int64]struct{}` instead.